ion/src/class/mod.rs

```diff
@@ -29,14 +29,18 @@ mod reflect;
 #[allow(dead_code)]
 #[derive(Debug)]
 pub struct ClassInfo {
-	class: &'static NativeClass,
-	constructor: *mut JSFunction,
-	prototype: *mut JSObject,
+	pub class: &'static NativeClass,
+	pub constructor: *mut JSFunction,
+	pub prototype: *mut JSObject,
 }
 
 pub trait ClassDefinition: NativeObject {
 	fn class() -> &'static NativeClass;
 
+	fn parent_class_info(_: &Context) -> Option<(&'static NativeClass, Local<*mut JSObject>)> {
+		None
+	}
+
```

Comment:
With the change in 8ecaf4c26929488dba151759b22d4ddc56ab2dda, `ClassDefinition::parent_proto` should be used instead, so this can be removed. Other uses/definitions will have to be changed accordingly.

Reply:
I'll rebase on that commit and fix the code accoredingly.